etc/config/typescript.amazon.properties

```diff
@@ -0,0 +1,18 @@
+# Default settings for TypeScript Compiler
+compilers=&tsc
+compilerType=typescript
+versionFlag=--version
+
+group.tsc.compilers=tsc_jit_gc:tsc_jit_nogc
+compiler.tsc_jit_gc.exe=/opt/compiler-explorer/tsc/bin/tsc
+compiler.tsc_jit_gc.sharedlibs=/opt/compiler-explorer/tsc/bin/lib/libTypeScriptRuntime.so
+compiler.tsc_jit_gc.name=TypeScript JIT (GC)
+compiler.tsc_jit_nogc.exe=/opt/compiler-explorer/tsc/bin/tsc
+compiler.tsc_jit_nogc.sharedlibs=
+compiler.tsc_jit_nogc.name=TypeScript JIT (No GC)
+
+defaultCompiler=tsc_jit_gc
```

Comment:
This will need to be updated too (to whatever you name the other, like `tsc_0.0.20` or whatever.